content/22.vaccines.md

```diff
@@ -192,16 +192,66 @@ As of December 11, 2020, the United States FDA approved this vaccine under an em
+This vaccine is vectored with a recombinant vesicular stomatitis virus and targeted against the ebola virus [doi:10.1016/j.cell.2020.03.011].
+Additionally, several Phase I and Phase II clinical trials for other vaccines are ongoing [@doi:10.1128/CVI.00298-16].
+This technology is currently being explored for its potential against numerous infectious diseases including malaria [@doi:10.1016/j.ymthe.2016.11.003; @doi:10.1016/j.vaccine.2006.07.035], ebola [@doi:10.1056/NEJMoa1410863; @doi:10.1093/infdis/jir349; @doi:10.1586/14760584.2014.885841]], human immunodeficiency virus (HIV) [@doi:10.1097/COH.0b013e328363d389; @doi:10.1016/j.virol.2014.09.004].
+The prior MERS and SARS initiated interest in the application of viral vector vaccines to human coronaviruses [@doi:10.3390/v12080861], but efforts to apply this technology to these pathogens had not yet led to a successful vaccine candidate.
+In the mid-to-late 00s, adenoviral vectored vaccines against SARS were found to induce SARS-CoV-specific IgA in the lungs of mice [@doi:10.1099/vir.0.81579-0], but were later found to offer incomplete protection in ferret models [@doi:10.1099/vir.0.2008/001891-0].
+Around 2014, interest arose in utilizing simian adenoviruses as vectors because of the reduced risk that human vaccine recipients would have prior exposure resulting in adaptive immunity [@doi:10.1038/mt.2013.284], and chimpanzee adenoviruses were explored as a potential vector in the development of a vaccine against _Middle East respiratory syndrome-related coronavirus_ (MERS-CoV) [@doi:10.3390/v12080861].
+In 2017, results were published from an initial investigation of two vaccine candidates against MERS-CoV containing the MERS-CoV S gene vectored with Chimpanzee Adenovirus, Oxford University #1 (ChAdOx1), a replication-deficient chimpanzee adenovirus [@doi:10.1016/j.vaccine.2017.05.032].
+This study reported that a candidate containing the complete spike protein sequence induced a stronger neutralizing antibody response in mice than candidates vectored with Modified Vaccinia virus Ankara.
+It was pursued in additional research, and in the summer of 2020 results of two studies were published.
+The first reported that a single dose of ChAdOx1 MERS induced an immune response and inhibited viral repliaction in macaques [@doi:10.1126/sciadv.aba8399].
+The second reported  promising results from a phase 1 trial that administered the vaccine to adults and measured safety/tolerability and immune response (as indicated through immune assays following vaccination) [@doi:10/ggtxgp].
+While not all of these results were available at the time that vaccine development programs against SARS-CoV-2 began, viral vector vaccines have also been explored against this hCoV.
+Here we review the current state of the art in recent efforts to develop viral vector vaccines against SARS-CoV-2.
+
+#### ChAdOx1 nCoV-19 (Astra Zeneca)
```

Comment:
```suggestion
#### ChAdOx1 nCoV-19 (AstraZeneca)
```

Reply:
HAHA I looked this up and swore it had a space and now I looked it up again and you're definitely right!!!!